src/services/api.js

```diff
@@ -1,36 +1,12 @@
 'use strict';
 
 const BASE_URL = 'https://api.themoviedb.org/3';
-const BASE_IMG_URL = 'http://image.tmdb.org/t/p/';
 const API_KEY = '1be31873de9aa764d36bafbd53752fba';
 
 export async function getTopRatedFilms() {
 
     let response = await fetch(`${BASE_URL}/movie/popular?api_key=${API_KEY}&language=en-US&page=1`);
     let json = await response.json();
```

Comment:
@Mmarvinn  Actually `response.json();` returning a `promise` that resolves to a JavaScript object. So will be  better to rename it to `data`

Reply:
thx.
fix